crates/bevy_animation/src/util.rs

```diff
@@ -0,0 +1,28 @@
+/// Fast approximated reciprocal square root.
+#[inline]
+pub(crate) fn approx_rsqrt(x: f32) -> f32 {
+    // Fall back to Quake 3 fast inverse sqrt, is has a higher error
+    // but still good enough and faster than `.sqrt().recip()`,
+    // implementation borrowed from Piston under the MIT License:
+    // [https://github.com/PistonDevelopers/skeletal_animation]
+    let x2: f32 = x * 0.5;
+    let mut y: f32 = x;
+
+    let mut i: i32 = y.to_bits() as i32;
+    i = 0x5f3759df - (i >> 1);
+    y = f32::from_bits(i as u32);
+
+    y = y * (1.5 - (x2 * y * y));
```

Comment:
```suggestion
    i = 0x5f1ffff9 - (i >> 1);
    y = f32::from_bits(i as u32);

    y = 0.703952253 * y * (2.38924456 - (x2 * y * y));
```

[Same performance, ~2.7x the accuracy](http://rrrola.wz.cz/inv_sqrt.html)